CoreScriptsRoot/ServerCoreScripts/ServerSocialScript.lua

```diff
@@ -38,7 +38,6 @@ local function decodeJSON(json)
 		return HttpService:JSONDecode(json)
 	end)
 	if not success then
-		print("decodeJSON() failed because", result, "Input:", json)
```

Comment:
Ya, we saw the issue in studio when we turned this on last week. ac3d1de06ecbe7111a31d12874d77738ca5fe3b8 fixes this by returning early if we're in studio, before any endpoints are hit.